src/Countries/Vietnam.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+use DateTimeZone;
+use IntlDateFormatter;
+use Spatie\Holidays\Calendars\ChineseCalendar;
+
+class Vietnam extends Country
+{
+    use ChineseCalendar;
+
+    public function countryCode(): string
+    {
+        return 'vn';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            // New Year's Day
```

Comment:
You can add English translations too instead of comments.

Reply:
And how can I do so?

Reply:
I have just merged master to this branch. I think I know what to do now.

Reply:
I have added translations in English for Vietnam. I would prefer to keep the comments for now, making the code more descriptive to everyone that way.